conference.js

```diff
@@ -175,6 +175,7 @@ let room;
  * lib-jitsi-meet to detect and invoke
  */
 window.JitsiMeetScreenObtainer = {
+    gDMSupported: true,
```

Comment:
See: https://github.com/jitsi/lib-jitsi-meet/pull/2632

Reply:
Are we ok with not including a version check here just in case? gDM has been supported since M72 but do we know if it will work as expected with electron starting 72? IMHO, it will be safer to stick to versions we know where it works as expected.

Reply:
Good point. I'll add an extra check to LJM.

This flag means JM has the necessary APIs to make it work via the iframe API but LJM should decide which flow to use.

Reply:
yeah makes sense to add it there.

Reply:
On a second thought, note that the new behavior will only happen in the Electron SDK 7 (currently unreleased).

I think supporting a > 2 year old electron with the new SDK is not in the cards.

Reply:
Argh! I did catch a problem! An old SDK would break with a new LJM + JM. I'll need to bring the config back and check them together.

Reply:
Why it is failing? 

I'm also wondering isn't it better to pass this to LJM instead of having it as a global? Or for some reason we can't do that?

Reply:
Pass how? During init?

Reply:
1) About the passing - I guess init of LJM works or maybe as part of createLocalTrack.

2) I'm sorry for the question, I'm almost certain I'm missing something here. I kind of don't understand why do we need this flag at all. Is it to accommodate use cases where somebody will use only  LJM without jitsi-meet? 

What I'm confused is that for all new versions after this PR we will have this flag set to true and in LJM we will only execute the code from the `if` branch where the flag is true and for the versions before this PR we will just have the old code. I mean since jitsi-meet is tied to a specific LJM version what is the point of the flag. Do we check it somewhere else?

Reply:
Let me take another look.

Reply:
Alright, I took another approach. See: https://github.com/jitsi/lib-jitsi-meet/pull/2634

Basically, we always try gDM first, and fallback to the previous flow if it's not supported (there is a specific exception that Electron throws).

Thus, we no longer need this flag (that's how I started but got lost along the way).

I also left comments both here and in LJM so we axe all the old code (and the global!) once an SDK supporting gDM has been out for a while.